calc.py

```diff
@@ -2,7 +2,19 @@
 import sys
 import time
 
+class NotSupportedVersionException(BaseException):
+    def __init__(self):
+        pass
 #all helper functions
+def versionCheck():
+    MiniumPythonVersion=(3, 10)    
```

Comment:
This might be a typo, do you mean "MinimumPythonVersion"?

Reply:
Yes, that was what I meant, but bad spelling.

Reply:
I need to fix it tomorrow. 11/2/2021

Reply:
👍

Reply:
Copy, on it asap